system/lib/wasmfs/syscalls.cpp

```diff
@@ -347,7 +347,7 @@ int __syscall_fstatat64(int dirfd, intptr_t path, intptr_t buf, int flags) {
   buffer->st_uid = 0;
   buffer->st_gid = 0;
   // Device ID (if special file) No meaning right now for Emscripten.
-  buffer->st_rdev = 1;
+  buffer->st_rdev = 0;
```

Comment:
is this change to match the old FS?

Reply:
Yes, for regular files at least. Special files have special st_rdev values in the legacy backend, but that didn't seem worth porting to WasmFS right now.